assets/js/admin/admin-scripts.js

```diff
@@ -1340,11 +1346,18 @@ jQuery.noConflict();
 						$media_modal_config = {
 							title   : give_vars.metabox_fields.file.button_title,
 							button  : {text: give_vars.metabox_fields.file.button_title},
-							multiple: false
+							multiple: false,
+                            contentUserSetting :false,
```

Comment:
@deepak can you explain this var?

Reply:
It is being used to show the uploader tab when **add media button** is click